src/pages/index.js

```diff
@@ -0,0 +1,363 @@
+  openModal(avatarSubmitModal);
+});
+// TODO - Don't forget form validation
+// TODO - Set up close button listener
+
+editFormElement.addEventListener("submit", handleEditFormSubmit);
+cardFormElement.addEventListener("submit", handleAddCardFormSubmit);
+avatarFormElement.addEventListener("submit", handleAvatarSumbit);
+
+function getcardElement(data) {
+  const cardElement = cardTemplate.content
+    .querySelector(".card")
+    .cloneNode(true);
+
+  const cardNameEl = cardElement.querySelector(".card__title");
+  const cardImageEl = cardElement.querySelector(".card__image");
+  const cardLikeBtn = cardElement.querySelector(".card__like-btn");
+  const cardDeleteBtn = cardElement.querySelector(".card__delete-btn");
+
+  // TODO - If the card is liked, set the active class on the card
+
+  cardNameEl.textContent = data.name;
+  cardImageEl.src = data.link;
+  cardImageEl.alt = data.name;
+
+  // Show whether card is liked or not
+
+  if (data.isLiked) {
+    cardLikeBtn.classList.add("card__like-btn_liked");
+  }
+
+  cardLikeBtn.addEventListener("click", (evt) => {
+    handleLike(cardElement, data); // Use API to update the card on the server
+    console.log("Card Liked");
+    cardLikeBtn.classList.toggle("card__like-btn_liked"); // Update teh UI
+  });
+
+  cardDeleteBtn.addEventListener("click", (evt) =>
+    handleDeleteCard(cardElement, data)
+  );
+
+  deleteForm.addEventListener("submit", (evt) => handleDeleteSumbit(evt));
+
+  // deleteBtn.addEventListener("submit", handleDeleteSumbit(evt) =>
+  //   handleDeleteCard(cardElement, data)
+  // );
+
+  cardImageEl.addEventListener("click", () => {
+    openModal(previewModal);
+    previewModalCaptionEl.textContent = data.name;
+    previewModalImageEl.src = data.link;
+    previewModalImageEl.alt = data.name;
+  });
+
+  // add other necesssary elements
+
+  return cardElement;
+}
+
+function handleAddCardFormSubmit(evt) {
+  evt.preventDefault();
+  const inputValues = { name: cardNameInput.value, link: cardLinkInput.value };
+  cardSubmitElement.textContent = "Loading...";
+
+  api
+    .addCard(inputValues)
+    .then((res) => {
+      const cardElement = getcardElement(res);
+      cardsList.prepend(cardElement);
+      closeModal(cardSubmitModal);
+    })
+    .catch((error) => {
+      console.log(error);
+    })
+
+    .finally(() => {
+      cardSubmitElement.textContent = "Save";
+    });
+}
+
+function handleAvatarSumbit(evt) {
+  // evt.preventDefault();
+
+  const submitLikeBtn = evt.submitter;
+  submitLikeBtn.textContent = "Saving...";
+
+  const inputValues = { avatar: avatarLinkInput.value };
+
+  // TODO - Call api, editavataruserInfo
+
+  // TODO - Prevent behavior
+  console.log(avatarLinkInput.value);
+  api
+    .editAvatarInfo(inputValues)
+    .then((data) => {
+      console.log(data.avatar);
+      setUserData(data);
+      closeModal(avatarModal);
+      evt.target.reset();
+      disableButton(cardSubmitElement, settings);
```

Comment:
`cardSubmitElement` is for the card form, but it's the `avatar` form submission here


You'll use the `submitButton` from line 337

```js
      disableButton(submitButton, settings);


```